docs/docs/references/preview-projects.mdx

```diff
@@ -0,0 +1,173 @@
+import CreatePreview from './assets/create-preview.jpg';
+import DeletePreview from './assets/delete-preview.jpg';
+
+# Preview Projects
+
+**Preview projects** are temporary Lightdash projects where you can safely experiment with your metrics, dimensions and charts without affecting your production project. 
```

Comment:
@TuringLovesDeathMetal I think this intro line should mention that charts and dashboards are copied from the production project - that's a key part of what makes them useful